projects/plugins/jetpack/extensions/blocks/videopress/edit-common-settings.native.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * WordPress dependencies
+ */
+import { ToggleControl, SelectControl } from '@wordpress/components';
+import { useMemo, useCallback } from '@wordpress/element';
+import { __, _x } from '@wordpress/i18n';
+
+const options = [
+	{ value: 'auto', label: _x( 'Auto', 'VideoPress preload setting', 'jetpack' ) },
+	{
+		value: 'metadata',
+		label: _x( 'Metadata', 'VideoPress preload setting', 'jetpack' ),
+	},
+	{ value: 'none', label: _x( 'None', 'VideoPress preload setting', 'jetpack' ) },
+];
+
+const VideoSettings = ( { setAttributes, attributes } ) => {
+	const { autoplay, controls, loop, muted, playsinline, preload } = attributes;
+
+	const toggleFactory = useMemo( () => {
+		const toggleAttribute = attribute => {
+			return newValue => {
+				setAttributes( { [ attribute ]: newValue } );
+			};
+		};
+
+		return {
+			autoplay: toggleAttribute( 'autoplay' ),
+			loop: toggleAttribute( 'loop' ),
+			muted: toggleAttribute( 'muted' ),
+			controls: toggleAttribute( 'controls' ),
+			playsinline: toggleAttribute( 'playsinline' ),
+		};
+	}, [ setAttributes ] );
+
+	const onChangePreload = useCallback(
+		value => {
+			setAttributes( { preload: value } );
+		},
+		[ setAttributes ]
+	);
+
+	return (
+		<>
+			<ToggleControl
+				__nextHasNoMarginBottom
+				label={ __( 'Autoplay', 'jetpack' ) }
+				onChange={ toggleFactory.autoplay }
+				checked={ !! autoplay }
+				help={ __( 'Autoplay may cause usability issues for some users', 'jetpack' ) }
```

Comment:
This string is being added to the localization strings files (https://github.com/wordpress-mobile/gutenberg-mobile/pull/6634/files#diff-d5d2ed0bfbc4117c67ed9c3822eda75a06c4cd100b1d51961a0df2bbe9718db5R57) because it's missing the final period. If we add it, we could reuse the translation from the Gutenberg GlotPress project.

Reply:
Added in https://github.com/Automattic/jetpack/pull/35637/commits/e88d3d4c562afec065c652de3756fe032fb308bc, thanks!

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/35637/commits/e88d3d4c562afec065c652de3756fe032fb308bc, thanks 🙇 !